crates/sui-protocol-config/src/lib.rs

```diff
@@ -1260,6 +1261,11 @@ impl ProtocolConfig {
                 cfg.feature_flags.simplified_unwrap_then_delete = true;
                 cfg
             }
+            17 => {
+                let mut cfg = Self::get_for_version_impl(version - 1, chain);
+                cfg.execution_version = Some(1);
```

Comment:
Does this need to be bumped every time sui-execution is cut? At a given point where we have v0, v1, and we make a cut v2, would this then be `Some(3)`? Will we always use the `latest`?

Reply:
We don't always have to use `latest` -- we are going to start off that way, to preserve our current behaviour of always developing on and releasing `latest`, but eventually (when the pace of development slows), we might want to move to a model where `latest` is only shipped to devnet and we choose when we ship it to testnet (and eventually mainnet) by doing an explicit cut.